lib/classes/CLI.js

```diff
@@ -63,30 +65,40 @@ class CLI {
     this.log = function() {};
   }
 
+  isHelpRequest(processedInput) {
+    const commands = processedInput.commands;
+    const options = processedInput.options;
+
+    if (options.version || options.v) return true;
```

Comment:
should we check `commands.length === 1 && commands[0] === 'version'` as well? I saw we're checking this below? maybe have a function `isVersionRequest` ?

Reply:
> Should we check `commands.length === 1 && commands[0] === 'version'` as well? 

We're checking that at L76.

> maybe have a function `isVersionRequest `

I think it's fine to cover both _help_ and _version_ under one function. I didn't try to name it as `isHelpOrVersionRequest`, for simplicity